newsletters/models.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils import timezone
+
+
+class department(models.Model):
+    '''
+    Stores the detail of department linked with its newsletters
+    '''
+    department = models.CharField(max_length=50,
+                                  blank=False,
```

Comment:
I don't think you need this explicitly.

Reply:
Okay sir. I'll shift this into newsletters model itself.